translator/trace/zipkin/status_code.go

```diff
@@ -131,3 +151,46 @@ func toInt32(i int) (int32, error) {
 	}
 	return 0, fmt.Errorf("outside of the int32 range")
 }
+
+func extractStatusFromError(attrib *tracepb.AttributeValue) *int32 {
+	// The status is stored with the "error" key
+	// See https://github.com/census-instrumentation/opencensus-go/blob/1eb9a13c7dd02141e065a665f6bf5c99a090a16a/exporter/zipkin/zipkin.go#L160-L165
+	var unknown int32 = 2
+
+	switch val := attrib.Value.(type) {
+	case *tracepb.AttributeValue_StringValue:
+		canonicalCodeStr := val.StringValue.GetValue()
+		if canonicalCodeStr == "" {
+			return &unknown
```

Comment:
Can we ignore empty value of this attribute and do not set `m.fromErrorTag.codePtr` based on it?

Reply:
fixed